### PR TITLE
Add CREATE & DROP TABLE support to Delta connector

### PR DESCRIPTION
**Pull request by agrawalreetika**
```
== RELEASE NOTES ==

Delta Changes
* Add CREATE TABLE support to Delta connector. 
* Add DROP TABLE support for the external table to Delta connector. 
```

### CODE REVIEW

**Comment by dnskr** [comment]
@vkorukanti Could you please review the PR or help us to find who can?

File: presto-delta/src/main/java/com/facebook/presto/delta/DeltaMetadata.java
Comment (nmahadevuni) [inline presto-delta/src/main/java/com/facebook/presto/delta/DeltaMetadata.java:130]: I believe the error message should say "Cannot create managed Delta table".

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java
Comment (nmahadevuni) [inline presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java:1361]: Does it matter that INSERT, UPDATE is not applicable to Delta tables? Should we just create a new method and remove those?

Comment (agrawalreetika) [reply presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java:1361]: Make sense to remove privileges for INSERT, UPDATE, and DELETE  for now Since these are not yet supported via connector. And this can be added later as the support is added

File: presto-delta/src/main/java/com/facebook/presto/delta/DeltaMetadata.java
Comment (nmahadevuni) [inline presto-delta/src/main/java/com/facebook/presto/delta/DeltaMetadata.java:132]: Do we support DROP table for Delta? Is DELETE permission required for that?

Comment (nmahadevuni) [reply presto-delta/src/main/java/com/facebook/presto/delta/DeltaMetadata.java:132]: Looks like DROP table support isn't there. Can you please add that too?

Comment (agrawalreetika) [reply presto-delta/src/main/java/com/facebook/presto/delta/DeltaMetadata.java:132]: DELETE permission would be required for DELETE operation on the table with SqlStandardAccessControl.
Currently, DROP Table is not supported in the connector. I will add support for dropping external table in this

**Comment by nmahadevuni** [comment]
I think we can squash commits into one. Please check the test failures.

**Comment by agrawalreetika** [comment]
> I think we can squash commits into one. Please check the test failures.

Actually, I kept 2 commits intentionally to segregate support. Sure I will check the test failures

**Review by nmahadevuni — APPROVED** [review]
LGTM

**Comment by yzhang1991** [comment]
Hi @prestodb/committers , this PR is reviewed and LGTM-ed by our internal reviewer. Can we have a committer to do the final review? Thank you!!

**Comment by yzhang1991** [comment]
Re-ping @prestodb/committers @tdcmeehan

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**
